[2024-09-10T07:31:02.147+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T07:31:02.180+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V3.second_task scheduled__2023-03-12T00:00:00+00:00 [queued]>
[2024-09-10T07:31:02.199+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V3.second_task scheduled__2023-03-12T00:00:00+00:00 [queued]>
[2024-09-10T07:31:02.200+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T07:31:02.255+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2023-03-12 00:00:00+00:00
[2024-09-10T07:31:02.271+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=18827) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T07:31:02.278+0000] {standard_task_runner.py:72} INFO - Started process 18842 to run task
[2024-09-10T07:31:02.279+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V3', 'second_task', 'scheduled__2023-03-12T00:00:00+00:00', '--job-id', '4229', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpbqfe7l1s']
[2024-09-10T07:31:02.282+0000] {standard_task_runner.py:105} INFO - Job 4229: Subtask second_task
[2024-09-10T07:31:02.378+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V3.second_task scheduled__2023-03-12T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T07:31:02.549+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-12T00:00:00+00:00'
[2024-09-10T07:31:02.550+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T07:31:02.588+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T07:31:02.590+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the second task!']
[2024-09-10T07:31:02.601+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T07:31:02.603+0000] {subprocess.py:93} INFO - hey, this is the second task!
[2024-09-10T07:31:02.604+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T07:31:02.636+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T07:31:02.637+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V3, task_id=second_task, run_id=scheduled__2023-03-12T00:00:00+00:00, execution_date=20230312T000000, start_date=20240910T073102, end_date=20240910T073102
[2024-09-10T07:31:02.702+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T07:31:02.713+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
